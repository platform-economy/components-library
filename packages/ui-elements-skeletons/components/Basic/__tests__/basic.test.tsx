import * as React from 'react';
import * as TestRenderer from 'react-test-renderer';
/*
A set of utilities for testing Styled Components with Jest.
This package improves the snapshot testing experience and
provides a brand new matcher to make expectations on the style rules.
*/
import 'jest-styled-components';

// Component
import { Basic } from '../index';

describe('ui-elements-skeletons', () => {
  describe('Components', () => {
    describe('Basic', () => {

      it('should render with default properties only', () => {
        const tree = TestRenderer
          .create(<Basic />)
          .toJSON();
        expect(tree).toMatchSnapshot();
      });

    });
  });
});
