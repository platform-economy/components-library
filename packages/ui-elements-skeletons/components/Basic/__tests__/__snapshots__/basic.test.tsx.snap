// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui-elements-skeletons Components Basic should render with default properties only 1`] = `
.c0 {
  background: #aaa;
  width: 400px;
  height: 40px;
  -webkit-animation: 2s ckwiNC linear infinite 0s;
  animation: 2s ckwiNC linear infinite 0s;
}

<div
  className="c0"
/>
`;
